viewing_party/party.py

```diff
@@ -1,23 +1,193 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    #if any parameter is empty return none
+    if title and genre and rating:
+        new_movie = {"title": title, "genre": genre, "rating": rating}
+        return new_movie
+    else:
+        return None
+
+
+
+def add_to_watched(user_data, movie):
+
+    #put movie inside value list of of movies
+    user_data["watched"].append(movie)
+
+    #return user data
+
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    
+    user_data["watchlist"].append(movie)
+
+    return user_data
+
+def watch_movie(user_data, title):

```

Comment:
Good work in using a ranged loop to avoid the dangers of modifying a data structure while iterating.

The one suggestion I have for this function is to use a more descriptive loop variable name instead of `i`. Good variable names improve the readability of our code by clearly communicating the purpose the data we use and their data type/structure.